readme.md

```diff
@@ -159,7 +159,7 @@ Jira notifications are messages sent to a Mattermost channel when a particular e
 
 Notifications and webhooks can be used together or you can opt for one of them.
 
-![This is a channel notification of a new bug that was created in Jira](https://github.com/mattermost/mattermost-plugin-jira/assets/74422101/e7020c3e-48f6-4825-8193-6a189f6c96eb)
+![This is a channel notification of a new bug that was created in Jira](./assets/ticket-created.png)
```

Comment:
Should this path point to the ``attach-from-post.png`` image file rather than ``ticket-created.png``?

Reply:
The image file committed to this PR is named ``attach-from-post.png``, but the code to display an image is a different file name. An incorrect file name would be one reason  we're seeing an image error placeholder in a rich preview.

Reply:
This PR was supposed to have two changes:

1. Change `attach-from-post.png` image to show the new location of the Jira actions buttons.
2. Update the image in the "What are notifications?" section to show a notification triggered from Jira.

Please see this conversation we had earlier: https://github.com/mattermost/mattermost-plugin-jira/pull/1005#discussion_r1427735820

Reply:
@mickmister I'm not sure why the rich diff is displaying a broken image; the image is being rendered correctly if you click on "View file"

Reply:
Thanks I think I understand now. IIUC we now have a fixed `attach-from-post.png`  file, but it's no longer being used as it was incorrectly being used?

I went ahead and put the image in the "attach message" section, and it looks good to me
 
![CleanShot 2024-01-24 at 14 36 16](https://github.com/mattermost/mattermost-plugin-jira/assets/6913320/5471b22f-d2fd-4e05-8941-ed45b492c3e0)